crates/hir/src/lib.rs

```diff
@@ -3138,6 +3135,19 @@ impl ConstParam {
     pub fn ty(self, db: &dyn HirDatabase) -> Type {
         Type::new(db, self.id.parent(), db.const_param_ty(self.id))
     }
+
+    pub fn default(self, db: &dyn HirDatabase) -> Option<String> {
```

Comment:
I'm not sure about the return type. It seems it should've been `hir::Const`, but for now I don't understand how to construct it conveniently from `chalk_id::Const`

Reply:
The result of this function is then converted to `ast` again? I think I would prefer avoiding this converting to and from string and returning ast node directly here. Is it possible?

Reply:
well, in some cases no intermediate AST appears, but you're right, it really should return `ast::ConstArg`. and yes, it's possible, I'll update it tomorrow.

thank you for the review!